src/util/read-package-json.js

```diff
@@ -0,0 +1,7 @@
+import fs from 'fs-promise';
+import path from 'path';
+
+export default (dir) => {
+  const packageData = fs.readFileSync(path.resolve(dir, 'package.json'), 'utf8');
```

Comment:
Seems to work now. I think it didn't like that it was one line before, or something.